Sources/ArgumentsUtils.swift

```diff
@@ -80,6 +80,8 @@ enum OutputDestination: ArgumentConvertible {
 enum TemplateError: Error, CustomStringConvertible {
   case namedTemplateNotFound(name: String)
   case templatePathNotFound(path: Path)
+  case noTemplateFound
```

Comment:
Because to me "noTemplateFound" looks like we provided a template option but couldn't find that template being asked